gulpfile.js

```diff
@@ -300,17 +299,16 @@ gulp.task('test-create-rollup', (cb) => {
 
 // This task updates the codegen_mappings.json file in sync with the azure-rest-api-specs public repo.
 gulp.task('sync-mappings-with-repo', (cb) => {
-  if (!specRepoDir) {
-    return cb(new Error('Please provide --spec-repo-dir <Absolute path to the directory where the azure-rest-api-specs is cloned.>'));
+  if (!specRoot) {
+    return cb(new Error('Please provide --spec-root <Absolute path to the local clone of the .../azure-rest-api-specs/specification repository folder.>'));
```

Comment:
I'm not sure I understand the usage of `...` here.

Reply:
I'll rewrite this comment.